apps/mark-scan/frontend/src/app_root.tsx

```diff
@@ -298,6 +300,35 @@ export function AppRoot({
     }
   }, [endCardlessVoterSessionMutateAsync]);
 
+  const handleKeyboardAudioEvent = useCallback(
```

Comment:
Thinking we should keep the keyboard controls consistent across all apps that might need audio -- these handlers could go further upstream to `libs/ui`.

Just added [`<KeyboardShortcutHandlers>`](https://github.com/votingworks/vxsuite/blob/main/libs/ui/src/ui_strings/keyboard_shortcut_handlers.tsx) with a focus on dev convenience, but that could be expanded to include these handlers to centralise things. WDYT?

Reply:
I like that idea. Looks like `KeyboardShortcutHandlers` is used in `UiStringsContextProvider` so should be de facto imported everywhere, but would it make sense to import in `app_base` or something more explicitly top-level?

Reply:
I think it could work well in `<AppBase>` if it's generalised beyond just language/audio shortcuts (as long as the relevant handlers are appropriately no-ops when there aren't language/audio contexts present), but that might also require rolling `UiStringsContextProvider` into `AppBase` to make sure `KeyboardShortcutHandlers` is rendered within context. Might be easier to reason about if  `UiStringsContextProvider` installs the keyboard shortcuts it needs

Reply:
Returned to rendering `KeyboardShortcutHandlers` in `UiStringsContextProvider` [f642f13](https://github.com/votingworks/vxsuite/pull/4501/commits/f642f13c320a4180cbb5858aa6c43747392618e0).

I think it's somewhat counterintuitive to render from there since the accessible controller and UI strings are separate concerns. But it feels more dangerous to scatter keypress handling across multiple places and possibly introduce conflicts.

Reply:
Got it - realizing now how closely the UI strings and screen reader are related, I think it makes sense to keep `KeyboardShortcutHandlers` within `libs/ui`. Plus relying on an implicit order for rendering feels like a big trap door.